config/database.ts

```diff
@@ -1,36 +1,33 @@
-const mongoose = require('mongoose');
-const { config, database, up } = require('migrate-mongo');
+import mongoose from 'mongoose';
+import { config, database, up } from 'migrate-mongo';
 
-const { Environment } = require('../services/environment');
+import { Environment } from '../services/environment';
 
-const initialize = async () => {
+export const initialize = async () => {
     try {
-        await mongoose.connect(Environment.MONGO_URL, {
+        await mongoose.connect(Environment.MONGO_URL!, {
```

Comment:
We should likely inject our own types for the env vars instead of doing non-null assertions on multiple lines. I will add a commit.

Reply:
Commit added, take a look and see if that seems like it will work.

Might add some complexity to local envs since all env vars are required now. Just set them to a random string if needed to avoid errors.

Reply:
I found out that these exclamation marks were once added in a very old version, but it wasn't needed anymore anyway because I cast the entire environment in some Zod validation, causing the code to only run if MONGO_URL is defined. So we can only just remove those :)